data/strings/strings.txt

```diff
@@ -15792,7 +15794,7 @@
   [editor_share_to_all_dialog_message_1]
     comment = Dialog before publishing the modifications to the public map.
     tags = android,ios
-    en = Make sure you did not enter any personal data.
+    en = Make sure you didn't enter any personal data.
```

Comment:
In standard written English, contractions should not be used.  So this change does not make sense.

Having read the text, I would say the point is not to avoid "personal" data but to avoid:

  - data that has privacy concerns and should not be published to the world, and
  - data that is not appropriate for OSM because it is not of general interest, separate from any privacy concerns